packages/design-system/src/components/Table/TableDataCell.jsx

```diff
@@ -31,24 +40,24 @@ export const TableDataCell = ({
 };
 
 TableDataCell.propTypes = {
-  /**
-   * The table data cell contents.
-   */
   children: PropTypes.node,
   /**
    * Additional classes to be added to the table cell element.
    */
   className: PropTypes.string,
   /**
-   * The `headers` attribute contains a list of `id` attributes of the associated data. If there is more than one id, they are separated by spaces.
+   * Table Data cells must have a `headers` attribute for stackable table.
+   * The `headers` attribute must link to a Table Header cell’s `id`.
+   * If there is more than one `id`, they are separated by spaces.
+   * For screen readers to create association between the header and data cells.
```

Comment:
A little more compact:

`TableDataCell` must define a `headers` prop for stackable tables. The `headers` prop is needed to associate header and data cells for screen readers. 

`headers` consist of a list of space-separated ids that each correspond to a `TableHeaderCell` element. [Read more on the `headers` attribute.](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/td#Attributes)